Bridg-It.cbp

```diff
@@ -1,42 +1,42 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Practic in fisierul asta nimic nu e schimbat, si totusi fiecare linie apare ca modificata. Asta e probabil pentru ca, fiind pe Windows, foloseste CRLF line endings. Poti sa citesti mai mult despre asta, daca vrei. Totusi, nu ne dorim ce se intampla acum
Te sfatuiesc sa bagi comenzile:

git config --global core.autocrlf false
git config --global core.eol lf

Apoi du-te in root-ul folderului BridgIt, si fa un fisier .gitattributes, si in el baga asta:

* text=auto

Apoi, ca sa normalizezi ending-urile la fisiere, ruleaza urmatoarele:

git rm --cached -rf .
git diff --cached --name-only -z | xargs -n 50 -0 git add -f

And it should work